geometry_msgs/PoseStamped pose
---
asctec_msgs/GPSData gpsdata
